gr-qtgui/lib/QFosphorColorMapper.cc

```diff
@@ -64,10 +67,13 @@ int QFosphorColorMapper::loadFromFile(QFile& file)
 
         /* Are we in a palette ? */
         if (kp < 0) {
-            sscanf(line.constData(), "%d %ms", &kp, &name);
+            ss.str() = line.toStdString();
+            ss >> kp;
+            ss >> name;
```

Comment:
Something about either of these, probably the name one since it's last, is causing an error. I didn't dig into exactly the error case, but making the other change and inserting a ss.clear(); after this line causes the color heat-map to appear like before

Reply:
Then let's do that!